hnn_core/gui/_viz_manager.py

```diff
@@ -114,7 +114,9 @@ def _update_ax(fig, ax, single_simulation, plot_type, plot_config):
 
     elif plot_type == 'PSD':
```

Comment:
Sorry I didn't get it. Do you mean to add docstrings about types that are allowed to be overlayed?

Reply:
Actually I'm realizing that this isn't necessary since you disable the add plot button. I originally thought there was logic in the code here specific to overlaid/non-overlaid plots but that isn't the case.